core/src/avm2/flv.rs

```diff
@@ -45,6 +48,18 @@ fn avm2_array_from_flv_values<'gc>(
         .into()
 }
 
+fn avm2_date_from_flv_date<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    unix_time: f64,
+    _local_offset: i16,
```

Comment:
is `_local_offset` actually supposed to be ignored? If not, can you add a TODO or a stub warning?

Reply:
The DateObject only stores the datetime in UTC. The AVM1 implementation also seems to ignore it